TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -329,10 +329,13 @@ Case cases[] = {
     Case("Test detach", test_detach),
     Case("Test multi call and time measure", test_multi_call_time),
     Case("Test multi ticker", test_multi_ticker),
+    /* Time drifting tests are unstable on CI. Disable them for now. */
+#if 0
```

Comment:
You could define a macro in order to be able to execute tests manually ?
#if TIME_DRIFT_TESTS

Reply:
How this macro will be defined?

Reply:
In your local mbed_app for ex,
or with -D compilation command

Reply:
in the test - if not defined, its disabled. Otherwise enable them - so can test locally

Reply:
Fixed.